#!/usr/bin/env python

PACKAGE = "steering"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("STEERING_LEFT_PWM", int_t, 0, "Parameter value to turn front wheels left", 0, 0, 1500)
gen.add("STEERING_RIGHT_PWM", int_t, 0, "Parameter value to turn front wheels right", 0, 0, 1500)

exit(gen.generate(PACKAGE, "config_steering", "Steering"))